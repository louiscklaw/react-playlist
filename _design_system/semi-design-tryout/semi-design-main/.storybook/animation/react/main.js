const config = require('../../base/base');

module.exports = {
  ...config,
  "stories": [
    '../../../packages/(semi-animation-react|semi-animation-styled)/_story/*.react.stories.(js|jsx)',
  ],
  typescript: {
    check: false,
    checkOptions: {}
  },
};
