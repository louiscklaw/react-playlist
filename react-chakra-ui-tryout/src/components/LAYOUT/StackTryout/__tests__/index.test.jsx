import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';

import { StackTryout } from '../';

describe('test StackTryout tryout', () => {
  test('helloworld', () => {
    const tree = renderer.create(<StackTryout />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
