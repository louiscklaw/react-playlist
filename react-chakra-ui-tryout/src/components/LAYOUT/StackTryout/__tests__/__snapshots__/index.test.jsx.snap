// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test StackTryout tryout helloworld 1`] = `
<div>
  StackTryout
  <div>
    <div
      className="chakra-stack css-1y5q8ss"
    >
      <div
        className="css-1fi7ndm"
      >
        1
      </div>
      <div
        className="css-heh9fa"
      >
        2
      </div>
      <div
        className="css-13xgoat"
      >
        3
      </div>
    </div>
  </div>
  <div>
    <div
      className="chakra-stack css-gw7h1"
    >
      <div
        className="css-1fi7ndm"
      >
        1
      </div>
      <div
        className="css-heh9fa"
      >
        2
      </div>
      <div
        className="css-13xgoat"
      >
        3
      </div>
    </div>
  </div>
  <div>
    <div
      className="chakra-stack css-zsoya5"
    >
      <div
        className="css-it9pqr"
      >
        1
      </div>
      <div
        className="chakra-stack__divider css-jutndq"
      />
      <div
        className="css-pfat6z"
      >
        2
      </div>
      <div
        className="chakra-stack__divider css-jutndq"
      />
      <div
        className="css-1v1y8x1"
      >
        3
      </div>
    </div>
  </div>
  <div>
    <div
      className="chakra-stack css-1clhiew"
    >
      <div
        className="css-1hmx26h"
      >
        <h2
          className="chakra-heading css-6dvxm2"
        >
          Plan Money
        </h2>
        <p
          className="chakra-text css-1fhgjcy"
        >
          The future can be even brighter but a goal without a plan is just a wish
        </p>
      </div>
      <div
        className="css-1hmx26h"
      >
        <h2
          className="chakra-heading css-6dvxm2"
        >
          Save Money
        </h2>
        <p
          className="chakra-text css-1fhgjcy"
        >
          You deserve good things. With a whooping 10-15% interest rate per annum, grow your savings on your own terms with our completely automated process
        </p>
      </div>
    </div>
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
</div>
`;
